### PR TITLE
ref(document-title): Add org and project name to the doc page title - Part 1

### PR DESCRIPTION
**Pull request by priscilawebdev**
Add orgSlug and projectSlug in the document Title if available.

it covers the following pages:

. Issue Details
. Release Details
. New Alert Rule
. Edit Alert
. Transaction Summary details
. Event Details

### CODE REVIEW

File: src/sentry/static/sentry/app/components/organizations/headerItem.tsx [hunk 1]
```diff
@@ -95,7 +96,10 @@ class HeaderItem extends React.Component<Props> {
           </ChevronWrapper>
         )}
         {locked && (
-          <Tooltip title={lockedMessage || 'This selection is locked'} position="bottom">
+          <Tooltip
+            title={lockedMessage || t('This selection is locked')}
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/organizations/headerItem.tsx:100]: minor fix :)

**Review by dashed — APPROVED** [review]

**Comment by benvinegar** [comment]
I have wanted us to do more around titles for ages. Thanks @priscilawebdev!

File: src/sentry/static/sentry/app/views/performance/vitalDetail/index.tsx [hunk 2]
```diff
@@ -108,8 +108,16 @@ class VitalDetail extends React.Component<Props, State> {
         ? undefined
         : (vitalNameQuery as WebVital);
 
+    // The vital details page will always have only one project selected
+    const project = projects.find(p => p.id === String(eventView.project));
+    const projectSlug = project?.slug;
```
Comment (Zylphrex) [inline src/sentry/static/sentry/app/views/performance/vitalDetail/index.tsx:113]: This is not true, the vital details page can be a cross project view.

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/performance/vitalDetail/index.tsx:113]: oh good catch...let me update it

**Pull request merged**